### PR TITLE
Remove code that serves VMO metrics for Prometheus

### PR DESCRIPTION
**Pull request by bobdonat**
Addresses Jira VZ-2455.

This pull request removes the code that was needed to get two VMO prometheus metrics which we don't scrape for.

### CODE REVIEW

File: pkg/vmo/controller.go [hunk 1]
```diff
@@ -412,15 +405,6 @@ func (c *Controller) syncHandlerStandardMode(vmo *vmcontrollerv1.VerrazzanoMonit
 		c.clusterInfo.KeycloakCABundle = clusterSecret.Data[constants.KeycloakCABundleData]
 	}
 
-	// If lock, controller will not sync/process the VMO env
-	labels := prometheus.Labels{"namespace": vmo.Namespace, "vmo_name": vmo.Name}
-	if vmo.Spec.Lock {
```
Comment (pfmackin) [inline pkg/vmo/controller.go:417]: Is lock only related to metrics?

Comment (bobdonat) [reply pkg/vmo/controller.go:417]: This is the code that references Lock.  I am not clear what it is used for.   I do not see the value being set anywhere in the VMO code.

			// If lock, controller will not sync/process the VMO env
	labels := prometheus.Labels{"namespace": vmo.Namespace, "vmo_name": vmo.Name}
	if vmo.Spec.Lock {
		zap.S().Infof("[%s/%s] Lock is set to true, this VMO env will not be synced/processed.", vmo.Name, vmo.Namespace)
		metrics.Lock.With(labels).Set(1)
		return nil
	}
	metrics.Lock.Delete(labels)

Comment (bobdonat) [reply pkg/vmo/controller.go:417]: Looks like it.  The field is not referenced anywhere in VMO.

Comment (bobdonat) [reply pkg/vmo/controller.go:417]: Do you think I should remove Lock from the CRD?

Comment (bobdonat) [reply pkg/vmo/controller.go:417]: fixed.  I had removed too much code.

**Review by pfmackin — APPROVED** [review]

**Pull request merged**